pkgs/by-name/re/redwax-tool/package.nix

```diff
@@ -1,19 +1,19 @@
 { lib, stdenv, fetchsvn
 , autoreconfHook, pkg-config, txt2man, which
-, openssl, apr, aprutil
+, openssl, nss, p11-kit, libical, ldns, libunbound, apr, aprutil
 }:
 
 stdenv.mkDerivation rec {
   pname = "redwax-tool";
-  version = "0.9.1";
+  version = "0.9.7";
 
   src = fetchsvn {
     url = "https://source.redwax.eu/svn/redwax/rt/redwax-tool/tags/redwax-tool-${version}/";
-    sha256 = "sha256-MWSB1AkkRS18UUHGq8EWv3OIXPSVHpmrdD5Eq1VdbkA=";
+    sha256 = "sha256-fill-me-in";
```

Comment:
Please get the right hash :)

Reply:
Had no clue as to where the hash came from, it appears to be a hash of a directory?

Reply:
The hash is used as parameter to fetchsvn. It is computed by checking out the redwax source at a specific version and hashing all files.

You can get the right hash by putting in `sha256 = lib.fakeHash;`, running the build and then copying the hash from the error message.

Reply:
Would it be possible to change this to the tarballs below https://archive.redwax.eu/dist/rt/ - this is the permanent home of every release, and you'll be relying on our SHA256 hashes / PGP signatures, rather than what svn gave you.

Reply:
Yes. Try
```
src = fetchurl {
  url = "https://archive.redwax.eu/dist/rt/redwax-tool-${version}/redwax-tool-${version}.tar.gz";
  hash = "sha256-GYxThKyvgeyQ1ksfRrqvDUC+8j/0WgskMgz2/4qN1R4=";
};
```